docs/DeveloperGuide.md

```diff
@@ -285,18 +285,27 @@ payment tracking and more in one efficient package, tailored specifically for fr
+|----------|--------------------|----------------------------------------------|--------------------------------------------------------------------|
+| `* * *`  | software developer | add clients                                  | update my client list                                              |
+| `* * *`  | software developer | remove clients                               | ensure my client list is not cluttered                             |
+| `* * *`  | software developer | view my clients and their details            | keep track of my clients and what tasks to do                      |
+| `* * *`  | software developer | update my clients' details                   | ensure my client list details are updated                          |
+| `* * *`  | software developer | view a client's payment status               | track who has paid and who still needs to make a payment           |
+| `* * *`  | software developer | view a client's project status               | monitor the progress of each client's project                      |
+| `* * *`  | software developer | view a client's status                       | easily identify active clients                                     |
+| `* *`    | software developer | filter my clients based on sections          | easily find specific groups of clients                             |
+| `* *`    | software developer | sort clients by category                     | keep my client list organised                                      |
+| `* *`    | software developer | set deadlines                                | ensure projects are completed before deadlines                     |
+| `* *`    | software developer | blacklist a client                           | avoid working with certain clients in future                       |
+| `* *`    | software developer | view clients on the blacklist                | view all the clients I intend to avoid                             |
+| `* *`    | software developer | whitelist a blacklisted client               | consider working with said client again                            |
+| `* *`    | software developer | view clients on the whitelist                | view all clients that are not on the blacklist                     |
+| `* *`    | software developer | archive a client                             | store information of old clients without cluttering my client list |
+| `* *`    | software developer | unarchive a client                           | bring the information of archived clients back to the client list  |
+| `* *`    | software developer | see reminders for the next earliest deadline | focus on the most urgent task first                                |
+| `*`      | software developer | keep notes for each client                   | take note of important details from meetings                       |
```

Comment:
If we include this here, do we need to talk about this in the user guide under a coming soon section?

Reply:
I'll update the UG